app/assets/stylesheets/application.scss

```diff
@@ -6,6 +6,7 @@
 @import "compass/css3";
 @import "bootstrap-sprockets";
 @import "bootstrap";
+@import "tether-shepherd/dist/css/shepherd-theme-arrows";
```

Comment:
would be nice to add a comment before this line to mark this import as a `node_modules` import. To make it easy when we start moving css to webpack.
I thought in something like:
```scss
// node_modules imports
@import "tether-shepherd/dist/css/shepherd-theme-arrows";
```